packages/e2e-tests/authenticated/logpoints-01.test.ts

```diff
@@ -25,6 +28,13 @@ async function load(page: Page, recordingId: string, apiKey: string) {
   await disableAllConsoleMessageTypes(page);
 }
 
+async function close(page: Page) {
+  const jsCoverage = await page.coverage.stopJSCoverage();
+
+  await addCoverageReport(jsCoverage, base.info());
+  await page.close();
```

Comment:
Too bad we have to remember to do this anywhere we use `context.newPage()` – should we abstract that too? (like we do with `startTest`?)

Reply:
We've only got two tests that try to do multi-page work atm, so I'm willing to let that be as-is for now.

Reply:
IMO this is worth addressing even while there are only a few tests that use the functionality, because it seems likely we will overlook this in the future. It also doesn't seem like it would require much effort to address now.